websites/A/Averge/presence.ts

```diff
@@ -0,0 +1,49 @@
+const presence1 = new Presence({
+  clientId: "1328861612735598664", // Deine Client ID
+}), browsingTimestamp1 = Math.floor(Date.now() / 1000);
+
+presence1.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    startTimestamp: browsingTimestamp1,
+    largeImageKey: "averge-eu-logo", // Dein Image-Key im PreMiD-Dashboard
```

Comment:
Use image url instead of discord asset key
```suggestion
    largeImageKey: "https://averge.eu/cdn/img/averge.png",
```